cmd/kubeadm/README.md

```diff
@@ -0,0 +1,63 @@
+# Kubernetes Cluster Bootstrap Made Easy
```

Comment:
This doc should be in the docs repo. None of the other commands have docs here. We are trying to consolidate where we put user docs.